docs/release-notes/NuGet-5.3.md

```diff
@@ -14,7 +14,7 @@ NuGet distribution vehicles:
 | NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
 |:---|:---|:---|
 | [**5.3.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.3](https://visualstudio.microsoft.com/downloads/) | [3.0.100](https://dotnet.microsoft.com/download/dotnet-core/3.0)<sup>1</sup> |
-
+| [**5.3.1**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.3.6](https://visualstudio.microsoft.com/downloads/) | [Future version: 3.0.10x](https://dotnet.microsoft.com/download/dotnet-core/3.0) |
```

Comment:
Change 3.0.10x to 3.0.101 according to Nick's reply: This will be in 3.0.101 that isn’t out yet.